modeling_tool_test.cc

```diff
@@ -0,0 +1,224 @@
+#include "modeling_tool.h"
+
+#include <gmock/gmock.h>
+#include "google/cloud/spanner/client.h"
+#include "google/cloud/spanner/mocks/mock_spanner_connection.h"
+#include <google/protobuf/text_format.h>
+
+namespace {
+using ::testing::_;
+using ::testing::Return;
+using ::testing::Field;
+using google::cloud::StatusOr;
```

Comment:
for using statement, it is better to unify the format. i.e. using ::google::cloud::StatusOr;